MyLibrary/BusinessLogic/DataProvider/IDataProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿// <copyright file="IDataProvider.cs" company="Peretiatko Anastasiia">
+// Copyright (c) Peretiatko Anastasiia. All rights reserved.
+// </copyright>
+
+using System.Collections.Generic;
+using BusinessLogic.LibraryModel;
+
+namespace BusinessLogic.DataProvider
+{
+    /// <summary>
+    /// Interface for data providers
+    /// </summary>
+    public interface IDataProvider
+    {
+        /// <summary>
+        /// Gets the books
+        /// </summary>
+        /// <value>
+        /// The books
+        /// </value>
+        List<Book> Books
```

Comment:
I think its more correct to use IEnumerable<> insted of List<> as the returning value.

Reply:
On this level I use the list of authors, I am not sure that is it justified to use IEnumerable on this stage